apps/sptribs/preview/aso/sptribs-postgres.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: dbforpostgresql.azure.com/v1api20210601
+kind: FlexibleServer
+metadata:
+  name: ${NAMESPACE}-${ENVIRONMENT}
+  namespace: ${NAMESPACE}
+spec:
+  version: "14"
+  network:
```

Comment:
you will not need network if the subnet and dnszone are same which is in the central file  - https://github.com/hmcts/cnp-flux-config/blob/master/apps/azureserviceoperator-system/resources/flexibleserver-postgres.yaml#L27-L31